### ISSUE TITLE
Forbidden (403) CSRF verification failed. Request aborted. after installing.

### ISSUE THREAD

**Issue by xeroticikot** [kobotoolbox/kobo-install#246]
**Description**
 
Installed with the run.py command, but installation couldn't finish, although the docker containers started & I've setup the nginx for my subdomain https://kf.codexcl.com. When I try to login with the credentials, it gives the Django error "Forbidden (403) CSRF verification failed. Request aborted. after installing.".  

**Steps to Reproduce**

**Expected behavior**

**Desktop**

 - OS:  Ubuntu 22.04
 - Python Version: 3.11
 - Docker Version: latest
 - Docker Compose Version: latest
 
**Additional context**

**Comment by paulmwatson** [comment]
Did you find a fix for this @xeroticikot ?

**Comment by muhammad-levi** [comment]
Did you find a fix for this @xeroticikot @paulmwatson ?

**Comment by muhammad-levi** [comment]
https://github.com/kobotoolbox/kpi/blob/main/kobo/settings/base.py#L56-L65

I think I managed to resolve the issue by passing environment variable `SESSION_COOKIE_DOMAIN` (I set the value as `.kobo.local`) into the KPI container. Then in my machine's `/etc/hosts` I added the subdomains `kobo.kobo.local`, `enketo.kobo.local`, and `kobocat.kobo.local` pointing to the IP of the machine running the KoboToolbox application (e.g. 127.0.0.1).

**Comment by paulmwatson** [comment]
Yes but I ended up using the advanced reverse proxy settings and not the default path.
